css/css-grid-1/implicit-grids/grid-support-grid-auto-columns-rows-002.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+    <head>
```

Comment:
Nit: We don't need `<html>`, `<head>`, `<body>` tags in this case.
We usually ignore them in new tests, I know some old tests have them (mostly because they were XHTML before and automatically converted into HTML).